tests/namespaced-admission-policy-tests.bats

```diff
@@ -2,43 +2,62 @@
 
 setup() {
 	load common.bash
-	wait_pods
+	wait_pods -n kubewarden
 }
 
 teardown_file() {
-	kubectl delete --wait --ignore-not-found pods --all
-	kubectl delete --wait --ignore-not-found -n kubewarden clusteradmissionpolicies --all
-	kubectl delete --wait --ignore-not-found -n kubewarden admissionpolicies --all
+	kubectl delete pods --all
+	kubectl delete -f $RESOURCES_DIR/namespaced-privileged-pod-policy.yaml --ignore-not-found
 }
 
-@test "[AdmissionPolicy end-to-end tests] Install AdmissionPolicy" {
+@test "[Namespaced AdmissionPolicy] Install AdmissionPolicy" {
 	apply_admission_policy $RESOURCES_DIR/namespaced-privileged-pod-policy.yaml
 }
 
-@test "[AdmissionPolicy tests] Launch a privileged pod in the default namespace should fail" {
-	kubectl_apply_should_fail $RESOURCES_DIR/violate-privileged-pod-policy.yaml
+@test "[Namespaced AdmissionPolicy] Privileged pod in the default namespace (should fail)" {
+	# Launch privileged pod (should fail)
+	run kubectl run nginx-privileged --image=nginx:alpine --privileged
+	assert_failure
+	assert_output --regexp '^Error.*: admission webhook.*denied the request.*cannot schedule privileged containers$'
+	run ! kubectl get pods nginx-privileged
```

Comment:
I think this line can be taken out: if kubectl run fails, there will be indeed no pod. Hence this line only tests `kubectl`, not our Kubewarden (this line appears in other similar tests too).